### PR TITLE
Adding new topics to the covid app tips

### PR DESCRIPTION
**Pull request by maxwellfundi**
@michaelclapham  I have now added the other topics for the up to topic 15.  Once you merge these the interns can now go ahead with the adding the content on the pages. Cheers.

### CODE REVIEW

File: src/app/topics/topics.page.html [hunk 1]
```diff
@@ -51,6 +51,13 @@
         <app-topic-6 *ngIf="topic.number===6"></app-topic-6>
         <app-topic-7 *ngIf="topic.number===7"></app-topic-7>
         <app-topic-8 *ngIf="topic.number===8"></app-topic-8>
+        <app-topic-8 *ngIf="topic.number===9"></app-topic-8>
```
Comment (michaelclapham) [inline src/app/topics/topics.page.html]: This needs to change to
`<app-topic-9 *ngIf="topic.number===9"></app-topic-9>`
and same for 10 through 15.

Comment (maxwellfundi) [reply src/app/topics/topics.page.html]: Omg, thank you! Am on it asap

Comment (maxwellfundi) [reply src/app/topics/topics.page.html]: @michaelclapham  I have now got the numbers right.

Comment (michaelclapham) [reply src/app/topics/topics.page.html]: Looks good

**Pull request merged**